### ISSUE TITLE
Self enclosing tag does not work in Markdown.

### ISSUE THREAD

**Issue by mauvm** [MalcolmK/dmp-references#12]
With `<reference .../>` I get:

![](http://s15.postimg.org/m66olfdiz/Screen_Shot_2015_03_03_at_4_27_02_PM.png)

---

But if I change the exact same code to `<reference ...></reference>`, it works as expected:

![](http://s30.postimg.org/giiwc44b5/Screen_Shot_2015_03_03_at_4_27_16_PM.png)

---

Not sure what's causing this.

**Comment by MalcolmK** [comment]
Me neither. But you consider this as a real problem? Cause I don't think it will be a problem to use `<reference></reference>` instead of `<reference />`.

And also, I don't think this is a bug in this plugin, but something of the third parties that are used for documark.

**Comment by mauvm** [comment]
The self closing tag is cleaner, because the element has no content. Content of the element should be displayed, which now also isn't the case (if I'm correct). For example:

``` markdown
Lorem ipsum dolar <reference ...>content here</reference> sit amet.
```

The `content here` part will be removed by this plugin, because the reference element itself is removed: https://github.com/MalcolmK/dmp-references/blob/master/index.js#L47

**Comment by MalcolmK** [comment]
@mauvm That is absolutely correct. Currently, I don't really feel the need to change this, but feel free to be my guest :)
